rules/job.go

```diff
@@ -58,16 +75,23 @@ outer:
 				continue
 			}
 
-			if matches(rule, component) {
+			if matches(rule, *component) {
 				logger.Infof("Rule %s matched component %s", rule, component)
 
 				incident := rule.Template
 				incident.IncidentRuleID = _rule.ID
-				incident.Status = "open"
+				incident.Status = api.IncidentStatusOpen
+				incident.Severity = "Low" // TODO: make this configurable
 				if incident.Type == "" {
 					incident.Type = api.IncidentTypeAvailability
 				}
 				incident.Title = component.Name + " is " + string(component.Status)
+
+				if _, ok := openIncidentsMap[_rule.ID.String()][incident.Title]; ok {
+					logger.Infof("Incident %s already exists", incident.Title)
+					continue // this rule already created this incident
+				}
+
 				if err := db.Gorm.Create(&incident).Error; err != nil {
```

Comment:
This auto created incident doesn't have a commander or a communicator. The ones created manually have both of them. Is this alright @moshloop ?

Reply:
Yes